lib/core/src/sdk.rs

```diff
@@ -1213,6 +1240,17 @@ impl LiquidSdk {
         let mrh_addr_hash = sha256::Hash::hash(mrh_addr_str.as_bytes());
         let mrh_addr_hash_sig = keypair.sign_schnorr(mrh_addr_hash.into());
 
+        let webhook = self.persister.get_webhook_url()?.map(|url| ReverseWebhook {
+            url,
+            hash_swap_id: Some(true),
+            status: Some(vec![
+                RevSwapStates::InvoiceExpired,
```

Comment:
Why do we need InvoiceExpired notification? Do we have any action item here?

Reply:
I think for the reverse swap all these failure statuses can be removed. The swap will be set to failed when the app is opened.

Reply:
I agree